airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json

```diff
@@ -14,6 +14,9 @@
     "KeywordId": {
       "type": ["null", "integer"]
     },
+    "Keyword": {
```

Comment:
additionalProperties: true

Should be added to the changed schema, to support backward capability.

Reply:
Hi @bazarnov, I just made the modification

Reply:
Please add it to the root level of the schema, not to the property

Reply:
Modified!